src/logger.py

```diff
@@ -0,0 +1,44 @@
+import logging
+from rich.logging import RichHandler
+
+FORMAT = "%(message)s"
+
+class Logger:
+    def __init__(self):
+        self.__FORMAT = "%(message)s"
+        logging.basicConfig(
+                level="NOTSET", format=FORMAT, datefmt="[%X]", handlers=[RichHandler(rich_tracebacks=True)]
```

Comment:
`rich_tracebacks` can cause a performance issue. Allow users to disable it via configuration

Reply:
done